18/README.md

```diff
@@ -0,0 +1,208 @@
+        },
+        "uri": "ni:///sha-256;b...123?fpt=ed25519-sha-256&cost=131072"
+      },
+      "public_keys": [
+        "Bob's public key"
+      ]
+    },
+    {
+      "amount": "1",
+      "condition": {
+        "details": {
+          "public_key": "Carly's public key",
+          "type": "ed25519-sha-256"
+        },
+        "uri": "ni:///sha-256;c...13C?fpt=ed25519-sha-256&cost=131072"
+      },
+      "public_keys": [
+        "Carly's public key"
+      ]
+    },
+    {
+      "amount": "1",
+      "condition": {
+        "details": {
+          "public_key": "Daniel's public key",
+          "type": "ed25519-sha-256"
+        },
+        "uri": "ni:///sha-256;c...7-8Y?fpt=ed25519-sha-256&cost=131072"
+      },
+      "public_keys": [
+        "Daniel's public key"
+      ]
+    }
+  ],
+  "version": "2.0"
+}
+```
+
+Now that the Election has been created, each Member can cast their vote. Bob starts:
+```json
+{
+  "asset": {
+    "id": "e...047"
+  },
+  "id": "a...123",
+  "inputs": [
+    {
+      "fulfillment": "f...qE8p",
+      "fulfills": {
+        "output_index": 1,
+        "transaction_id": "e...047"
+      },
+      "owners_before": [
+        "Bob's public key"
+      ]
+    }
+  ],
+  "metadata": {
+    "type": "vote"
+  },
+  "operation": "TRANSFER",
+  "outputs": [
+    {
+      "amount": "1",
+      "condition": {
+        "details": {
+          "public_key": "Election Address",
+          "type": "ed25519-sha-256"
+        },
+        "uri": "ni:///sha-256;b...123?fpt=ed25519-sha-256&cost=131072"
+      },
+      "public_keys": [
+        "Election Address"
+      ]
+    }
+  ]
+  "version": "2.0"
+}
+```
+
+## Rationale
+<!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+Another idea to implement this protocol is to use a multi signature transaction.
+
+## Backwards Compatibility
+<!--All BEPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The BEP must explain how the author proposes to deal with these incompatibilities. BEP submissions without a sufficient backwards compatibility treatise may be rejected outright.-->
+This BEP is fully backwards compatible.
+
+## Implementation
+<!--The implementations must be completed before any BEP is given status "stable", but it need not be completed before the BEP is accepted. While there is merit to the approach of reaching consensus on the BEP and rationale before writing code, the principle of "rough consensus and running code" is still useful when it comes to resolving many discussions of API details.-->
+
```

Comment:
You can say that this section can be filled in once there _is_ an implementation.